### DISCUSSION TITLE
Having trouble installing girara/zathura on macOS Monterey

### DISCUSSION THREAD

**Discussion by Dorima1** [Homebrew/discussions#3530] [Getting started]
I am receiving the following error running both 'brew install girara --HEAD' and 'brew install zathura --HEAD'. I am not sure how to fix the error I posted in the following screenshot.
<img width="599" alt="Screen Shot 2022-08-14 at 10 57 31 PM" src="https://user-images.githubusercontent.com/83615435/184569791-8b84bfa2-2b0e-4857-98f9-c9575174ea77.png">

**Comment by gromgit** [comment]
You're installing from the `zegervdv/zathura` tap, so you should report an issue there instead of Homebrew.

**Reply by SMillerDev** [reply]
As is also clearly described in the screenshot

**Comment by carlocab** [comment]
`girara` needs to either depend on `json-c` or pass `-Djson=disabled` to `meson`.

This needs to be fixed at https://github.com/zegervdv/homebrew-zathura. Homebrew does not manage that repository.
